radix-engine/src/blueprints/account/package.rs

```diff
@@ -245,11 +245,14 @@ impl AccountNativePackage {
                 let input: VirtualLazyLoadInput = input.as_typed().map_err(|e| {
                     RuntimeError::SystemUpstreamError(SystemUpstreamError::InputDecodeError(e))
                 })?;
-                let rtn = AccountBlueprint::create_virtual_ecdsa_256k1(input.id, api)?;
+
+                let public_key_hash =
```

Comment:
Minor: can we move the conversion from `Vec<u8>` to `PublicKeyHash` into the `create_virtual` function as the input for virtualization should be a generic byte array (only the function should know what it actually represents).

Reply:
Can do. I viewed the AccountBlueprint as wanting to talk in terms of abstractions it understands (ie `PublicKeyHash`) and felt that handling NodeIds and how they translate to hashes was the remit of this virtualization layer? But happy to move it :)